crates/astria-conductor/src/reader.rs

```diff
@@ -232,7 +232,7 @@ impl Reader {
                 if let Err(e) = self
                     .block_verifier
                     .validate_rollup_data(
-                        &data.data.block_hash,
+                        data.data.block_hash.as_bytes(),
```

Comment:
Does it make sense to redefine this function to take the hash instead of a byte slice now that all types have been updated?

Reply:
Yes. This was an easy change.